### DISCUSSION TITLE
Password for shell

### DISCUSSION THREAD

**Discussion by zzolax** [zephyrproject-rtos/zephyr#72591] [General] [answered]
Hello,
I use NRF52840 with NUS shell over bluetooth, for security reason is it possible to enable login to shell?

**Comment by ndrs-pst** [comment]
@zzolax please take a look at [here](https://docs.zephyrproject.org/3.6.0/services/shell/index.html#obscured-input-feature).

**Reply by zzolax** [reply, marked as answer]
Thank you!
